Library/Formula/ansible.rb

```diff
@@ -22,6 +22,7 @@ class Ansible < Formula
   depends_on :python if MacOS.version <= :snow_leopard
   depends_on "libyaml"
   depends_on "openssl"
+  depends_on "libffi" if OS.linux?
```

Comment:
The `ffi` formula installs `ffi.h` in a subdirectory, which seems to be the default behaviour of `libffi`.
```
$ brew list libffi |grep ffi.h
/gsc/btl/linuxbrew/Cellar/libffi/3.0.13/lib/libffi-3.0.13/include/ffi.h
$ ls `brew --prefix libffi`/lib/libffi-3.0.13/include
ffi.h  ffitarget.h
```